components/dashboard/src/settings/PersonalAccessTokens.tsx

```diff
@@ -156,7 +156,26 @@ function ListAccessTokensView() {
                             </a>
                         </PillLabel>
                     </h3>
-                    <h2 className="text-gray-500">Create or regenerate access tokens.</h2>
+                    <h2 className="text-gray-500">
+                        Create or regenerate access tokens.{" "}
+                        <a
+                            className="gp-link"
+                            href="https://www.gitpod.io/docs/configure/user-settings/access-tokens"
+                            target="_blank"
+                            rel="noreferrer"
+                        >
+                            Learn more
+                        </a>
+                        &nbsp;&middot;&nbsp;
+                        <a
+                            href="https://github.com/gitpod-io/gitpod/issues/15433"
+                            target="_blank"
+                            rel="noreferrer"
+                            className=" text-sm gp-link"
```

Comment:
**issue**: I skipped mentioning above in https://github.com/gitpod-io/gitpod/pull/15450#discussion_r1061691330 that the links should use the same font size when they are placed next to each other like this.

```suggestion
                            className="gp-link"
```

| BEFORE | AFTER |
|-|-|
| <img width="508" alt="b" src="https://user-images.githubusercontent.com/120486/210746105-58dbfe93-8bec-4bcd-9b9c-858649fa6ac2.png"> | <img width="508" alt="a" src="https://user-images.githubusercontent.com/120486/210746108-84b11858-5107-46f3-bf8a-4ffba85a701b.png"> |